python/paddle/fluid/layers/nn.py

```diff
@@ -5827,6 +5828,47 @@ def rank_loss(label, left, right, name=None):
     return out
 
 
+def margin_rank_loss(label, left, right, margin=0.1, name=None):
+    """
+    **Margin Rank loss layer for rank problem**
```

Comment:
This doesn't seems to explain what does this operator do?

Reply:
I enriched the doc in next commit